test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -715,6 +725,26 @@ public void getParticipantWithoutHistories() {
         assertTrue(participant.getConsentHistories().keySet().isEmpty());
     }
     
+    @Test
+    public void getParticipantByAccountId() {
+        AccountId accountId = AccountId.forEmail(STUDY.getIdentifier(), "email@email.com");
+        when(accountDao.getAccount(accountId)).thenReturn(account);
+        
+        StudyParticipant participant = participantService.getParticipant(STUDY, accountId, true);
+        
+        assertNotNull(participant);
+        verify(accountDao).getAccount(accountId);
+    }
+    
+    @Test(expected = EntityNotFoundException.class)
+    public void getParticipantByAccountIdThrowsException() {
+        AccountId accountId = AccountId.forEmail(STUDY.getIdentifier(), "email@email.com");
+        
+        participantService.getParticipant(STUDY, accountId, true);
+        
+        verify(accountDao).getAccount(accountId);
```

Comment:
Minor: This verify will never be called, since participantService.getParticipant() throws.

Reply:
removed